spec/models/customstates.rb

```diff
@@ -1,52 +1,47 @@
 # -*- encoding : utf-8 -*-
-module InfoRequestCustomStates
-
-  def self.included(base)
-    base.extend(ClassMethods)
-  end
+class InfoRequestThemeStates
 
   # Mixin methods for InfoRequest
-  def theme_calculate_status
-    return 'waiting_classification' if self.awaiting_description
-    waiting_response = self.described_state == "waiting_response" || self.described_state == "deadline_extended"
-    return self.described_state unless waiting_response
-    if self.described_state == 'deadline_extended'
+  def calculate_status(info_request)
+    return 'waiting_classification' if info_request.awaiting_description
+    waiting_response = info_request.described_state == "waiting_response" || info_request.described_state == "deadline_extended"
+    return info_request.described_state unless waiting_response
+    if info_request.described_state == 'deadline_extended'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.